Firestore/Source/Local/FSTLevelDB.h

```diff
@@ -96,6 +97,8 @@ NS_ASSUME_NONNULL_BEGIN
 /** The native db pointer, allocated during start. */
 @property(nonatomic, assign, readonly) std::shared_ptr<leveldb::DB> ptr;
 
+@property(nonatomic, readonly) firebase::firestore::local::LevelDbTransaction *current_transaction;
```

Comment:
Objective-C style would name this property `currentTransaction`.

Reply:
Done.